piqe_utils/api/logger.py

```diff
@@ -0,0 +1,95 @@
+""" Module for getting logger object
+
+###############################################################################
+# How to use :
+###############################################################################
+
+---------------------------
+For tests dir :
+---------------------------
+1) Import piqe_logger class in main conftest.py of project and pass
+__loggername__ as an argument.
+
+2) Create a fixture with (scope='session', autouse=True)
+
+i.e.
+
+import pytest
+from piqe_utils.api.logger import PiqeLogger
+from piqe_utils import __loggername__
+
+
+@pytest.fixture(scope='session', autouse=True)
+def setup_logger():
+    _ = PiqeLogger(__loggername__)
+
+-------------------------------------
+For Lib dir :
+-------------------------------------
+1) logger = logging.getLogger(__loggername__)
+
+i.e.
+
+import logging
+from piqe_utils import __loggername__
+logger = logging.getLogger(__loggername__)
+
+class ModuleA(object):
+    def method_a1(self):
+        logger.debug('Message from ModuleA, method_a1')
+
+    def method_a2(self):
+        logger.debug('Message from ModuleA, method_a2')
+
+                   OR
+
+For Logger in main conftest.py (project level) :
+2) Create logger at project level (main) conftest.py with (scope='session', autouse=True)
```

Comment:
We should probably be more specific with this example. Maybe write a quick example of a fixture that initializes this logger for writing tests. I would also be specific that this example is specific with using pytest.

Reply:
> We should probably be more specific with this example. Maybe write a quick example of a fixture that initializes this logger for writing tests. I would also be specific that this example is specific with using pytest.

I have modified the documentation now with the specific example of how to use it with pytest. Please review again and let me know if any other changes are required.